# Patch made by Salonia Matteo <saloniamatteo@protonmail.com>
# Made for saluto-lightdm-theme-dwm
# Apply to saluto/src/lightdm.js
--- lightdm.js	2020-11-22 09:32:31.708516415 +0100
+++ lightdm.js	2020-11-22 09:32:07.781741362 +0100
@@ -47,7 +47,11 @@
             {
                 name: 'xmonad',
                 key: 'xmonad'
-            }
+            },
+            {
+		name: 'dwm',
+		key: 'dwm'
+	    }
         ],
         users: [{
             display_name: 'Marcus Tullius Cicero',
